src/app/index.config.coffee

```diff
@@ -59,10 +59,11 @@ angular.module 'mnoEnterpriseAngular'
     }
 
     availableLocales = _.map(I18N_CONFIG.available_locales, 'id')
+    preferredLocale = I18N_CONFIG.preferred_locale.slice(0,2)
```

Comment:
What is the problem when using the full code?

Reply:
@adamaziz15 ping

Reply:
@ouranos Ah, angular-translate does not support the full codes by default. See: https://angular-translate.github.io/docs/#/guide/09_language-negotiation